examples/pt2/torch_export_aot_compile/resnet18_torch_export.py

```diff
@@ -9,7 +9,15 @@
 model.eval()
 
 with torch.no_grad():
-    device = "cuda" if torch.cuda.is_available() else "cpu"
+    if torch.cuda.is_available():
+        device = "cuda"
+    else:
+        device = "cpu"
+        # The below config is needed for max batch_size = 16
+        # https://github.com/pytorch/pytorch/pull/116152
```

Comment:
Can we check the batch_size input to reflect the max batch_size limitation ?

Reply:
Thanks..Updated the comments. There is no limitation on batch_size.